lib/screens/add_transaction_screen.dart

```diff
@@ -199,6 +199,23 @@ class TransactionsState extends State<AddTransactionScreen> {
                   ),
                 ],
               ),
+              Row(
+                children: <Widget> [  
+                  Expanded(
+                      child: CheckboxListTile(
+                        title: Text("Repeat"),
```

Comment:
Text aus dem localisationprovider ziehen und in die JSON schreiben.
Außerdem einen Textstyle hinzufügen damit die Farbe sich mit dem Theme ändert